packages/react-server/core/ReactServerAgent/Request.js

```diff
@@ -123,6 +124,15 @@ Request.prototype.toJSON = function(){
 	};
 }
 
+// Enable CORS
+Request.prototype.withCredentials = function (on) {
+	if (on === undefined) {
+		on = true;
+	}
+	this._withCredentials = on;
```

Comment:
If `this._withCredentials` is expected to be a boolean, I'd recommend changing this to: `this._withCredentials = !!on;`